src/classes/CW_MessagingApprovalControllerTest.cls

```diff
@@ -0,0 +1,165 @@
+/**
+* File Name   :    CW_MessagingApprovalControllerTest
+* Description :    approver can approve/reject
+*
+* Modification Log
+* ============================================================================
+* Ver Date       Author         Modification
+* --- ---------- -------------- --------------------------
+* 0.1 20/08/2014Suneetha Jasthi     B-122329  Mailer/teaser/SMS messaging approval processes
+*
+*/
+
+
+@isTest
```

Comment:
Classes and triggers should use the `UpperCamelCase` naming convention. I suggest to rename accordingly.